packages/mobile/src/backup/utils.ts

```diff
@@ -105,6 +107,17 @@ export async function getStoredMnemonic(): Promise<string | null> {
   }
 }
 
+export function useAccountKey() {
+  const [accountKey, setAccountKey] = useState<string | null>(null)
+
+  useAsync(async () => {
+    const mnemonic = accountKey || (await getStoredMnemonic())
+    setAccountKey(mnemonic)
+  }, [])
+
+  return accountKey
+}
```

Comment:
I think `useState` is unnecessary here, can be simplified to:
```suggestion
export function useAccountKey() {
  asyncAccountKey = useAsync(getStoredMnemonic, [])
  return asyncAccountKey.result;
}
```